src/sentry/integrations/github/client.py

```diff
@@ -499,27 +496,7 @@ def _populate_tree(
         repo_files = self.get_cached_repo_files(
             full_name, branch, only_use_cache=only_use_cache, cache_seconds=cache_seconds
         )
-        return RepoTree(Repo(full_name, branch), repo_files)
-
-    def get_repositories(self, fetch_max_pages: bool = False) -> Sequence[Any]:
```

Comment:
i think we still might want this here, but just return `repos` without any extra processing

Reply:
I have updated this function. Please have another look.